### DISCUSSION TITLE
What was the former name of 🇿🇼?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1034] [Galaxy Trivia] [answered]
**Bonus:** How many neighboring countries does it have today?

**Comment by peterbaccaro** [comment]
The former name of Zimbabwe was **Southern Rhodesia**. It was a British self-governing colony established in 1923. The name Southern Rhodesia was used until the country's Unilateral Declaration of Independence (UDI) from Britain on November 11, 1965. After this declaration, the country was led by an administration that was not recognized by the international community, and it continued to be known as Southern Rhodesia until 1980. On April 18, 1980, the country gained internationally recognized independence, and its name was changed to Zimbabwe. The new name was inspired by the historic archaeological site of Great Zimbabwe, which was the capital of the Kingdom of Zimbabwe, an ancient civilization that flourished in the region from the 11th to the 15th century. The name change marked the end of white-minority rule and the beginning of Zimbabwe as an independent nation.

Zimbabwe shares borders with **four** neighbouring countries:
- South Africa to the south,
- Botswana to the west and southwest,
- Mozambique to the east and northeast,
- Zambia to the northwest.

**Comment by thuongtruong1009** [comment, marked as answer]
The former name of 🇿🇼 (Zimbabwe) was "Southern Rhodesia."  Zimbabwe has the following neighboring countries:

1. Zambia to the northwest
2. Mozambique to the northeast and east
3. South Africa to the south
4. Botswana to the southwest

Please note that geopolitical situations can change over time, so it's always a good idea to verify this information with up-to-date sources.

**Comment by NomanGhiasy** [comment]
Southern Rhodesia
